books/RayTracingInOneWeekend.html

```diff
@@ -483,23 +483,15 @@
 <div class='together'>
 Let’s add a single object to our ray tracer. People often use spheres in ray tracers because
 calculating whether a ray hits a sphere is pretty straightforward. Recall that the equation for a
-sphere centered at the origin of radius $R$ is $x^2 + y^2 + z^2 = R^2$. The way you can read that
-equation is “for any $(x, y, z)$, if $x^2 + y^2 + z^2 = R^2$ then $(x,y,z)$ is on the sphere, and
-otherwise it is not”. It gets uglier if the sphere center is at
-$(\mathbf{c}_x, \mathbf{c}_y, \mathbf{c}_z)$:
+sphere centered at the origin is $x^2 + y^2 + z^2 = R^2$.
```

Comment:
I think the commentary about interpreting the value of the left-hand side is to set up understanding for later code where we distinguish between points _inside_ the sphere and points _on_ the sphere.